metricbeat/module/system/_meta/config.full.yml

```diff
@@ -16,7 +16,7 @@
     - filesystem
 
     # File system summary stats
-    #- fsstat
+    - fsstat
```

Comment:
Just making sure. We had this disabled before because it wasn't working on Windows and now it is?